src/main/java/com/negotium/negotiumapp/service/UserService.java

```diff
@@ -24,24 +33,23 @@ public UserService(PasswordEncoder passwordEncoder) {
         this.passwordEncoder = passwordEncoder;
     }
 
-
     @Autowired
-    public void setUserRepository(UserRepository userRepository) {
+    public void UserService(UserRepository userRepository) {
```

Comment:
Pytanie: Czy użycie osobnych konstruktorów dla poszczególnych pól było konieczne? Czy niewystarczyłby wieloargumentowy konstruktor?

Reply:
Nie było koniecznie, jest to raczej kwestia czytelności lub indywidualnych upodobań

Reply:
Coś mi się wydaje, że problem z encoderem wynika z tego, że tylko jeden konstruktor może mieć oznaczenia @Autowired.